src/SDKs/ServiceBus/data-plane/Directory.Build.props

```diff
@@ -28,4 +22,14 @@
     -->
     <DefineConstants Condition="'$(OS)' == 'Windows_NT'">$(DefineConstants);WINDOWS</DefineConstants>
   </PropertyGroup>
+
+  <!-- 
+    For Track 1, override the engineering system signing key and use the existing key for
+    Azure messaging client libraries.
+  -->
+  <PropertyGroup Condition="'$(SignAssembly)' != 'false'">
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)assets\azure-messaging.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>false</DelaySign>
```

Comment:
Please set PublicSign=false as well as it is a good idea to disable that for any places we have a full key.